### PR TITLE
Figure-eight knot knowledge

### PR DESCRIPTION
**Pull request by wilson-ca-mac-com**
https://github.com/instructlab/taxonomy/issues/823

### CODE REVIEW

File: knowledge/knots/figure_eight_knot/qna.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+created_by:
```
Comment (bjhargrave) [inline knowledge/knots/figure_eight_knot/qna.yaml:1]: The value must be your github id.

File: knowledge/knots/figure_eight_knot/qna.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  question: What type  of knot is the figure-eight knot?
+- answer: |The figure-eight knot is used in both sailing and rock climbing as a method of stopping ropes from running out of retaining devices.
+  question: Where and how are figure-eight knots used?
+- answer: |The figure-eight knot is also called the figure-of-eight knot or Flemish knot.
+  question: What other names is the figure-eight knot known by?
+- answer: |The different types of figure-eight knots are the figure-eight loop, figure-eight bend, offset figure-eight bend, and stein knot.
+  question: What are the different types of figure-eight knots?
+- answer: |The figure-eight loop is used like an overhand loop knot. This type of knot can be used in prusik climbing when used in conjunction with a climbing harness, a climbing rope, and locking carabiner designed for climbing, to ascend or descend with minimal equipment and effort. 
+  question: How is the figure-eight loop used?
+task_description: ' '
```
Comment (bjhargrave) [inline knowledge/knots/figure_eight_knot/qna.yaml:14]: A complete task description must be provided.

**Review by bjhargrave — CHANGES_REQUESTED** [review]
Please correct the lint issues and address the review comments.

**Comment by jjasghar** [comment]
Hi! :wave:
With the new update and release of 0.18.0, our [base knowledge submissions](https://github.com/instructlab/taxonomy/?tab=readme-ov-file#knowledge-yaml-examples) have drastically changed. This has given us an opportunity to significantly improve the accuracy of our fine-tuned models, which is a major win for our community!
But alas, that has caused some changes that this PR isn't able to continue forward. We greatly appreciate the work you've put into this, and wish we could accept it but the amount of work to get it to be `version: 3` is unrealistic for us to ask.
If you _do_ want to do this, please don't hesitate to take the `qna.yaml` update it, and build it out. Otherwise, we hope you come back to our community with a new submission and help build out our Open Source community-driven LLM.

If you have any questions directly, please don't hesitate to comment on the PR or email me at awesomeATinstructlab.ai (yes, really), and I'll do my best to answer your questions.

**Pull request closed without merging**